serveradmin/resources/management/commands/clear_migration_log.py

```diff
@@ -0,0 +1,50 @@
+"""Serveradmin - igvm_migration_log cleanup
+
+We are using igvm_migration_log for calculating the overall CPU usage of a
+Hypervisor. This is obsolete after the next prime time because the cpu_util_pct
+will be correct again. This task will get executed once in the morning to
+clean up the log entries from the previous day.
+
+Copyright (c) 2020 InnoGames GmbH
+"""
+from datetime import datetime, timedelta
+from django.core.management.base import BaseCommand
+
+from adminapi.dataset import Query
+from adminapi.filters import Not, Empty
+
+
+class Command(BaseCommand):
+    help = 'Deletes igvm_migration_log entries from previous day'
+
+    def handle(self, *args, **options):
+        """Entry point for command
+
+        get all hypervisors which have an entry in igvm_migration_log
+        and delete all entries which are from the previous day.
+        """
+
+        all_hypervisors = Query({'igvm_migration_log': Not(Empty()),
+                                 'servertype': 'hypervisor'},
+                           ['igvm_migration_log', 'hostname'])
+
+        current_timestamp = datetime.utcnow()
+        hour = current_timestamp.hour
+
+        for hypervisor in all_hypervisors:
+
+            for entry in hypervisor['igvm_migration_log']:
+                timestamp = entry.split()[0]
+
+                dt_object_sa = datetime.fromtimestamp(int(timestamp))
```

Comment:
When working with dates and times, try to be consistent with the timezones. `fromtimestamp` without the timezone parameter will convert the timestamp to the local time without being timezone aware.
With `utcnow` you would now compare a timezone-naive (which will be treated as the local time, although it is utc time) with a timezone-naive local datetime. If the local time of the system is not also utc, this will behave wrongly.
With `now` (without specifying the utc timezone) you would now compare a timezone-naive local time with a timezone-naive conversion of a timestamp to local time, which should behave correctly
With `now` and utc timezone, you would now compare a timezone-aware utc time with a timezone-naive conversion of a timestamp to local time, which is not even possible.
With `now` and utc timezone and `fromtimestamp(timestamp, tz=timezone.utc)` you would now compare a timezone-aware utc time with a timezone-aware conversion of a timestamp to utc time, which is behaving correctly.